src/app/game/word-ladder/play/page.module.css

```
@@ -0,0 +1,606 @@
+  color: #ea580c;
+  transform: scale(1.05);
+}
+
+.tileWon {
+  background: linear-gradient(135deg, #22c55e, #16a34a) !important;
+  border-color: #16a34a !important;
+  color: white !important;
+  animation: popIn 0.4s ease forwards;
+}
+
+@keyframes popIn {
```

Stylelint (17.10.0)

[error] Expected keyframe name "popIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)